crypto/fipsmodule/sha/internal.h

```diff
@@ -62,10 +77,29 @@ void sha512_block_data_order(uint64_t *state, const uint8_t *in,
                              size_t num_blocks);
 #endif
 
+// SHA3_224 writes the digest of |len| bytes from |data| to |out| and returns |out|. 
+// There must be at least |SHA3_224_DIGEST_LENGTH| bytes of space in |out|.
+// On failure SHA3_224 returns NULL.
```

Comment:
```suggestion
// On failure |SHA3_224| returns NULL.
```
Since it is not the first word in the paragraph.